ls1intum/Artemis
# `Assessment`: Integrate reviewer information into list of complaints

### Checklist
#### General


#### Client

### Motivation and Context
Closes #5800
Closes #2906

### Description
The list of complaints now additionally shows the name of the complaint reviewer. This information is only visible for instructors to keep complaints double-blind for tutors. To avoid too much clutter in the table, it now only shows the name of the assessor and reviewer, not the login.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Tutor
- 1 Course with existing complaints about multiple exercises

1. Open the List of Complaints:
- For the whole course (Course Management -> "Complaints" at the title of the chart)
- For one exercise (Course Management -> Assessment Dashboard -> Select an exercise -> Select complaints in the table)
- [Tutors]: For own corrections (Course Management -> Assessment Dashboard -> Select Complaints in the table)
2. Check that Instructors see the information about students and reviewers, while tutors stay double-blind
3. Check that sorting the table works

#### Exam Mode Testing
Prerequisites:
- 1 Instructor
- 1 Tutor
- 1 Exam with existing complaints about multiple exercises

1. Open the List of Complaints:
- For the whole exam (Course Management -> Exams -> Exam -> Assessment -> Select complaints in the table)
- For one exercise (Course Management ->Exams -> Exam -> Assessment -> Select an exercise -> Select complaints in the table)
2. Check that Instructors see the information about students and reviewers, while tutors stay double-blind
3. Check that sorting the table works

### Review Progress






#### Code Review
#### Manual Tests
#### Exam Mode Test

### Screenshots
